python/gpusim_createdb.py

```diff
@@ -95,7 +97,10 @@ def writeData(self, qds):
 
 
 def main():
+    global dview
     args = parse_args()
+    if args.singleThreaded:
+        dview = None
     qf = QtCore.QFile(args.outputfile)
```

Comment:
blah. I guess I would have preferred having the ipyparallel try import block assign ipp = None in the except, and then move the dview extraction to main

Reply:
Good suggestion